site/src/AppRouter.tsx

```diff
@@ -413,6 +412,7 @@ export const AppRouter: FC = () => {
             </Route>
 
             {/* Pages that don't have the dashboard layout */}
+            <Route path="/:username/:workspace" element={<WorkspacePage />} />
```

Comment:
seemed like the old spot for this made more sense, next to its sub-route?

Reply:
I'm removing this from the DashboardLayout because it is not going to be any more in the very near future. It also made it easier for me to remove the deployment banner from the bottom.